projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/src/components/editor-placeholder.tsx

```diff
@@ -0,0 +1,32 @@
+import { classNames } from '../utils';
+import { CustomLoadingSpinner } from './custom-loading-spinner';
+
+export const EditorPlaceholder = ( { onClick, loading } ) => {
+	return (
+		<div
+			className="verbum-editor-wrapper"
+			role="presentation"
+			onClick={ onClick }
+			onKeyDown={ onClick }
+		>
+			<div
+				class={ classNames( 'editor__main loading-placeholder', {
+					loading,
+				} ) }
+			>
+				<div
+					class="block-list-appender block-editor-block-list__layout"
+					style={ { padding: '10px 20px' } }
+				>
+					{ loading ? (
+						<CustomLoadingSpinner />
```

Comment:
Added custom spinner to remove dependency on `@wordpress/components` and reduce project size.

Reply:
that's great!